module/VuFind/src/VuFind/View/Helper/Root/ExplainElement.php

```diff
@@ -61,9 +61,14 @@ public function __invoke($explainElement, $decimalPlaces)
         $fieldValue = $explainElement['fieldValue'] ?? false;
         $shortLabel = '';
         if (is_array($fieldName) && is_array($fieldValue) && count($fieldName) == count($fieldValue)) {
-            $shortLabel = 'Synonym [' . implode(', ', array_map(function ($name, $value) {
-                return $name . '(' . $value . ')';
-            }, $fieldName, $fieldValue)) . ']';
+            $shortLabel = $view->transEsc('Synonym') . '['
```

Comment:
I don't think escaping here is the right thing. That should be something for the template. I'd prefer the translation to be done there as well, but in this case it's not worth the effort.

Reply:
fixed